rollup.config.js

```diff
@@ -0,0 +1,49 @@
+import { readdirSync } from 'fs';
+import { resolve } from 'path';
+import babel from 'rollup-plugin-babel';
+
+const listOfEnties = readdirSync(resolve(__dirname, './build'));
+
+export default listOfEnties.map(entry => {
+  let output;
+  switch (entry) {
+    case 'index.js':
+      output = [
+        {
+          file: resolve(__dirname, './publish/dist/index.cjs.js'),
+          format: 'cjs'
```

Comment:
I though tree shaking works only with ES modules?

Reply:
I think the we still need to provide a entry file for who's build tools doesn't support es module syntax(e.g. webpack1, jest or use in Node.js directly), and then people can still use the same import useage (`import { A, B } from 'mdi-react'`) which is more convenient than `import A from 'mdi-react/A';import B from 'mdi-react/B'`. And people can also use [babel-plugin-import](https://www.npmjs.com/package/babel-plugin-import) make tree-shaking available even with commonjs built.